src/course-unit/course-xblock/CourseXBlock.jsx

```diff
@@ -47,10 +55,33 @@ const CourseXBlock = ({
   const intl = useIntl();
   const xblockIFrameHtmlAndResources = useSelector(getXBlockIFrameHtmlAndResources);
   const xblockInstanceHtmlAndResources = find(xblockIFrameHtmlAndResources, { xblockId: id });
+  const [showLegacyEditModal, toggleLegacyEditModal] = useState(false);
+  const xblockLegacyEditModalRef = useRef(null);
+
   const {
     canCopy, canDelete, canDuplicate, canManageAccess, canManageTags, canMove,
   } = actions;
 
+  useOverflowControl(`.${XBLOCK_EDIT_MODAL_CLASS_NAME}`);
```

Comment:
```suggestion
  const editModalElement = document.querySelector(XBLOCK_EDIT_MODAL_SELECTOR);
  useOverflowControl(editModalElement);
```

I think that it will be better that hook receives an element instead of selector to make sure that we can pass either element from the DOM or React `ref` like `ref.current`